tools/download_test.go

```diff
@@ -40,7 +40,7 @@ func TestDownloadCorrectPlatform(t *testing.T) {
 		{"darwin", "amd64", "x86_64-apple-darwin"},
 		{"darwin", "arm64", "arm64-apple-darwin"},
 		{"windows", "386", "i686-mingw32"},
-		{"windows", "amd64", "i686-mingw32"},
+		{"windows", "amd64", "x86_64-mingw32"},
```

Comment:
The Windows 32 version must be downloaded if the Windows 64 is missing.

We could implement a compatibility logic similar to the CLI:
https://github.com/arduino/arduino-cli/blob/master/arduino/cores/tools.go#L129-L216

I don't know how much work is to import directly `cores.Tools` from arduino-cli (so we can implement the same logic also for macosx), probably it's not worth it...

Anyway, IMHO, at least the compatibility rule for Windows 64 -> Windows 32 must be implemented.